dace/transformation/helpers.py

```diff
@@ -382,9 +382,8 @@ def unsqueeze_memlet(internal_memlet: Memlet,
         # Special case: If internal memlet is one element and the top
         # memlet uses all its dimensions, ignore the internal element
         # TODO: There must be a better solution
-        if (len(internal_subset) == 1 and ones == list(range(len(shape)))
```

Comment:
what happened to the state_fission fix?

Reply:
??

Reply:
https://github.com/spcl/dace/pull/645/files#diff-f76940db8b32cd3a0807a9d0b3b9fa8aa845dd60ff5c91cb6b6f292db860373b

Reply:
Thanks, I included it!